packages/python/m/github/actions/action_file.py

```diff
@@ -0,0 +1,144 @@
+from m.core import yaml
+from m.pydantic import KebabModel
+from pydantic import BaseModel
+from pydantic.fields import PydanticUndefined
+
+from .actions import Action
+from .misc import MetadataOutput
+
+TEMPLATE = """\
+# AUTOGENERATED FILE - Do not update by hand!
+# Edit {py_path}/{module_name}.py
+#   then run `m github actions [python_file]` to update
+
+name: {name}
+description: {description}
+
+inputs:{inputs}
+outputs:{outputs}
+runs:
+  using: composite
+  steps:
+    {steps}
+"""
+EMPTY_DICT = ' {}'  # noqa: P103 - not a format string
```

Comment:
is this supposed to have the space here?

Reply:
Good point, may be better to do

```py
EMPTY_DICT = '{}'
```

then

```py
f'prop: {EMPTY_DICT}'
```